package.json

```diff
@@ -1,5 +1,17 @@
 {
+  "name": "opg-lpa",
+  "description": "The Office of the Public Guardian Lasting Power of Attorney online service: Managed by opg-org-infra &amp; Terraform.",
+  "version": "1.0.0",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs",
+    "test": "tests"
+  },
+  "scripts": {
+    "lint:check": "eslint service-front/assets/js cypress/e2e/common"
+  },
   "devDependencies": {
+    "@babel/eslint-parser": "^7.19.1",
```

Comment:
Not required, as we don't have any code which needs babel. The default eslint parser is adequate for our purposes.

Reply:
Done